.github/labeler.json

```diff
@@ -1,53 +1,65 @@
 {
   "labels": {
-    "area: examples": ["examples/**"],
-    "area: documentation": ["docs/**", "errors/**"],
     "area: create-next-app": ["packages/create-next-app/**"],
-    "type: next": [
-      "packages/next/**",
-      "packages/react-dev-overlay/**",
-      "packages/react-refresh-utils/**",
-      "packages/next-codemod/**",
-      "packages/eslint-plugin-next/**",
-      "packages/eslint-config-next/**",
-      "packages/next-env/**",
-      "packages/next-swc/**"
+    "area: documentation": ["docs/**", "errors/**"],
+    "area: examples": ["examples/**"],
+    "area: next/image": [
```

Comment:
Any additional areas we add here we should handle in the changelog generation https://github.com/vercel/next.js/blob/111ef3951cf13e9eb758ace222265fdd3f883d6e/release.js#L3